packages/backend/src/accounting/psql/ledger-transfer/index.ts

```diff
@@ -254,7 +253,7 @@ async function validateBalances(
       transferAmount: amount
     })
   ) {
-    return TransferError.InsufficientDebitBalance
```

Comment:
So interestingly, the `peer liquidity withdrawal` and `asset liquidity withdrawal` resolver tests in `liqudity.test.ts` expected a `InsufficientBalance` error instead of a `InsufficientDebitBalance` here.

_However_, `InsufficientDebitBalance` seems to make more sense here, since we don't have enough debits to withdraw from the settlement/debit account. Not sure what is best here, `InsufficientBalance` does keep it simple